content/22.vaccines-trad.md

```diff
@@ -36,38 +36,34 @@ The COVID-19 pandemic has generated several unique circumstances relative to pas
 As early as September 2020, there were over 180 vaccine candidates against SARS-CoV-2 in development, many of which used more traditional vaccine technologies [@doi:10.1038/s41586-020-2798-3].
-However, media attention has largely focused on vaccine development platforms that have produced vaccine candidates using previously unemployed technologies, especially mRNA vaccines.
+However, media attention has largely focused on vaccine development platforms that have produced vaccine candidates using previously unemployed technologies, especially mRNA vaccines due to their rapid deployment.
 We review vaccine technologies used for SARS-CoV-2 in two parts: here, the application of traditional vaccine development platforms to SARS-CoV-2, and separately, omics-based approaches [@individual-vaccines-novel].
 
 Vaccine development using well-established technologies is important for a global perspective on COVID-19.
 As of {{owid_most_recent_date}}, {{viper_vaccine_counts}} SARS-CoV-2 vaccines have been approved. Data tracking the administration of {{owid_vaccine_counts}} vaccines throughout the world indicates that {{owid_total_vaccinations}} doses have been administered across {{owid_total_countries}} countries [@doi:10.1038/s41562-021-01122-8].
 Of these, {{viper_num_approved_subunit}} were developed using subunit and {{viper_num_approved_whole_virus}} using whole-virus approaches, which do not require information about the viral genome.
 Many vaccines are available but in only a subset of countries, and the types of vaccines available varies widely throughout the world, as the process of developing and deploying a vaccine is complex and often requires coordination between government, industry, academia, and philanthropic entities [@doi:10.1126/science.abc5312].
 
-![
-**Structure of the SARS-CoV-2 virus.**
-The development of vaccines depends on the immune system recognizing the virus.
-Here, the structure of SARS-CoV-2 is represented both in the abstract and against a visualization of the virion.
-The abstracted visualization was made using BioRender [@url:https://biorender.com] and the microscopy was conducted by the National Institute of Allergy and Infectious Diseases [@url:https://www.niaid.nih.gov/news-events/novel-coronavirus-sarscov2-images].<!--To Do: Was a template used?-->
-](images/SARS_CoV_2.png){#fig:virus secno=1}
-
+<!--Do we need a wider definition of VE? For example - https://www.who.int/news-room/feature-stories/detail/vaccine-efficacy-effectiveness-and-protection. I ask because there has been blurred lines between whether we are trying to prevent disease or infection… which are not mutually exclusive-->
 One of the metrics through which a vaccine's success is measured is efficacy.
 The vaccine efficacy (VE) of a vaccine seeks to quantify the extent to which a vaccine protects against the virus.
 However, protection can mean different things in different contexts.
```

Comment:
```suggestion
Additionally, protection can mean different things in different contexts.
In general, the goal of a vaccine is to prevent disease, especially severe disease, rather than infection itself.
```